addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -44,7 +44,7 @@
                                         <br t-if="not subtitle_last"/>
                                     </t>
                                 </td>
-                                <td t-else=""><span style="font-weight:bold;" t-out="record_name"/><br/></td>
+                                <td t-else=""><span style="font-weight:bold;" t-out="record_name or (message.record_name and message.record_name.replace('/','-')) or ''"/><br/></td>
```

Comment:
Missed this one too :eyes: 

Otherwise lgtm, too bad we have to propagate "force_record_name" everywhere but I guess the only alternative would be a crappy context key

Reply:
At some point I think we will add kwargs for "small tweaks", and just document it somewhere. Keep "main API parameters" and voila.